Assets/com.fluid.behavior-tree/Editor/BehaviorTree/Printer/NodePrintController.cs

```diff
@@ -40,25 +39,38 @@ public class NodePrintController {
             var prevBackgroundColor = GUI.backgroundColor;
             
             var rect = new Rect(
-                _box.GlobalPositionX + _box.PaddingX, 
+                _box.GlobalPositionX + _box.PaddingX,
                 _box.GlobalPositionY + _box.PaddingY,
-                _box.Width - _box.PaddingX, 
+                _box.Width - _box.PaddingX,
                 _box.Height - _box.PaddingY);
 
             if (_node.Task.HasBeenActive) {
                 GUI.backgroundColor = _faders.BackgroundFader.CurrentColor;
-                GUI.Box(rect, GUIContent.none, Styles.BoxActive.Style);
+
+                if (EditorGUIUtility.isProSkin) {
+                    GUI.Box(rect, GUIContent.none, new GUIStyle(DisplaySettings.darkModeBoxStyle));
+                }
+                else {
+                    GUI.Box(rect, GUIContent.none, new GUIStyle(DisplaySettings.lightModeBoxStyle));
+                }
+
                 GUI.backgroundColor = prevBackgroundColor;
-                
                 PrintLastStatus(rect);
-            } else {
-                GUI.Box(rect, GUIContent.none, Styles.BoxInactive.Style);
             }
-            
+            else {
+                if (EditorGUIUtility.isProSkin) {
```

Comment:
Move if block to a static `DisplaySettings.GetBoxInactiveStyle()` method that can support caching